README.md

```diff
@@ -103,6 +125,26 @@ A job can be inserted into the Repo directly by constructing a job with the `new
 |> MyApp.Repo.insert()
 ```
 
+A job can be inserted with optional params:
+
+- `:schedule` : runs the job at the given `%DateTime{}`. The default value is `DateTime.utc_now()`.
+- `:max_attempts` : the maximum attempts for this job. The default value is `5`.
+- `:priority` : the priority of this work, the default value is `0`. Increase this value to decrease priority.
```

Comment:
I like the description of the priority field in delayed job: ":priority (number): lower numbers run first; default is 0"

Reply:
@mbuhot Done!